src/platform/update/new_release_monitor.cpp

```diff
@@ -35,7 +35,9 @@ namespace
 {
 constexpr auto timeout = std::chrono::minutes(1);
 constexpr auto json_tag_name = "version";
-constexpr auto json_html_url = "release_url";
+constexpr auto json_html_url = "download_url";
```

Comment:
Why this? It's `release_url` that has more details about the release, no?

Reply:
@Saviq, I thought we wanted to drive folks to `multipass.run` and not `github`.

Reply:
Not for this usecase I don't think, since they won't get any more details on https://multipass.run…